normalize-git-url.js

```diff
@@ -18,7 +18,16 @@ module.exports = function normalize (u) {
 
   var returnedUrl
   if (parsed.pathname.match(/\/?:/)) {
-    returnedUrl = u.replace(/^(?:git\+)?ssh:\/\//, '').replace(/#[^#]*$/, '')
+    if (u.match(/^git\+https?/) && parsed.pathname.match(/\/?:[^0-9]/)) {
+      returnedUrl = u.replace(/^(.*:[^:]+):(.*)/, '$1/$2')
+      console.warn('deprecated: ' + u + ' is using an' +
```

Comment:
Yeah, if we really want to warn on this, that'd be better as an addition to `normalize-package-data`, but I don't think warning on this is important enough to delay this patch.